langchain/package.json

```diff
@@ -609,7 +613,14 @@
     "@gomomento/sdk": "^1.51.1",
     "@gomomento/sdk-core": "^1.51.1",
     "@jest/globals": "^29.5.0",
-    "@langchain/cohere": "^0.0.8",
+    "@langchain/anthropic": "^0.2.8",
```

Comment:
Should these be `workspace:*`?

Reply:
Would be better, but this will still default to the local code, and updating to `workspace:*` will require a significant update to the latest/lowest dep CI test which I don't really want to do atm